source/how-tos/support_ticket.rst

```diff
@@ -84,23 +84,23 @@ the support ticket form fields can be arranged or changed as required.
 
 .. note::
   
-  Please note that an ``email`` field must be present in the form for the support ticket feature to be funtional.
+  Please note that an ``email`` field must be present in the form for the support ticket feature to be functional.
 
-Custom fields can be added, but they will require a custom email template to make use of the field values to generate the email body.
-Drop the template file into ``/etc/ood/config/apps/dashboard/views/support_ticket/email`` folder and reference it
-in the configuration. This template file can be any format Rails recognizes for emails, usually the ``.txt.erb`` extension.
+Custom fields can be added, but they will require a custom email template to make use of the provided values in the email body.
+Override the default email template with your own by dropping a file named ``_email.text.erb`` into the folder
+``/etc/ood/config/apps/dashboard/views/support_ticket/email/``
 
-.. warning::
+.. tip::
 
- Rails expects files to be prefixed with an underscore. For example if you configured ``my_email_template``
- the filename should be ``_my_email_template.text.erb``.
+  To test the custom template in the development or test environments,
+  please enable the external configuration environment variable: ``OOD_LOAD_EXTERNAL_CONFIG=true``.
+  We recommned setting environment vaariables in the ``/etc/ood/config/apps/dashboard/env`` file.
```

Comment:
`OOD_LOAD_EXTERNAL_CONFIG` is sort of an undocumented development feature. In production they'll always load external configs. It's only there so that developers can bypass the system configs.